commons/commons-rest-client/src/main/java/com/thinkbiganalytics/rest/JerseyClientConfig.java

```diff
@@ -242,4 +233,35 @@ public String getKeystoreType() {
     public void setKeystoreType(String keystoreType) {
         this.keystoreType = keystoreType;
     }
+
+    @Override
+    public String toString() {
+        return "JerseyClientConfig{" +
+               "host='" + host + '\'' +
+               ", port=" + port +
+               ", username='" + username + '\'' +
+               ", password='XXX'" +
+               ", https=" + https +
+               ", keystoreOnClasspath=" + keystoreOnClasspath +
+               ", keystorePath='" + keystorePath + '\'' +
+               ", keystorePassword='YYY'" +
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)